app/lambda/lambda_function.py

```diff
@@ -1,29 +1,39 @@
 import boto3
```

Comment:
If boto3 is not required, please remove the reference

Reply:
Done. https://github.com/aws-samples/medical-transcription-analysis/blob/691348b78d89ba8652fa177e9a002b4466b0ef6b/app/lambda/lambda_function.py#L1-L2